syntax = "proto3";

package health;

// Request message for Health
message HealthRequest {
}
